ocs_ci/ocs/bucket_utils.py

```diff
@@ -1833,7 +1862,11 @@ def s3_copy_object(s3_obj, bucketname, source, object_key, metadata=None):
     metadata = {} if metadata is None else metadata
 
     return s3_obj.s3_client.copy_object(
-        Bucket=bucketname, CopySource=source, Key=object_key, Metadata=metadata
+        Bucket=bucketname,
+        CopySource=source,
+        Key=object_key,
+        Metadata=metadata,
+        MetadataDirective=metadata_directive,
```

Comment:
We can pass `**kwargs` and we dont need to worry about addition of any extra parameters